config/argocd/templates/0200-argocd.yaml

```diff
@@ -445,63 +451,9 @@ spec:
 
         hs.status = "Progressing"
         return hs
-    operators.coreos.com/CatalogSource:
-      health.lua: |
-        hs = {}
-        if obj.status ~= nil then
-          if obj.status.connectionState ~= nil then
-            if obj.status.connectionState.lastObservedState ~= nil then
-              if obj.status.connectionState.lastObservedState == "READY" then
-                hs.status = "Healthy"
-                return hs
-              end
-            end
-          end
-        end
-
-        hs.status = "Progressing"
-        hs.message = "Unknown"
-        return hs
-    operators.coreos.com/InstallPlan:
-      health.lua: |
-        hs = {}
-        if obj.status ~= nil then
-          if obj.status.phase ~= nil then
-            hs.message = obj.status.phase
-            if obj.status.phase == "Complete" then
-              hs.status = "Healthy"
-            else
-              hs.status = "Progressing"
-            end
-            return hs
-          end
-        end
-
-        hs.status = "Progressing"
-        hs.message = "Unknown"
-        return hs
-    operators.coreos.com/Subscription:
-      health.lua: |
-        hs = {}
-        if obj.status ~= nil then
-          if obj.status.installedCSV ~= nil then
-            hs.status = "Healthy"
-            hs.message = "CSV Installed"
-            if obj.status.state ~= nil then
-              hs.message = obj.status.state
-            end
-            return hs
-          end
-        end
-
```

Comment:
The recent releases of OpenShift GitOps already have health checks for these OLM resources.